flytekit/clis/sdk_in_container/package.py

```diff
@@ -113,6 +136,12 @@ def package(
     object contains the WorkflowTemplate, along with the relevant tasks for that workflow.
     This serialization step will set the name of the tasks to the fully qualified name of the task function.
     """
+    if copy == CopyFileDetection.TEMP_NO_COPY:
+        raise ValueError("--copy none doesn't need to be specified, package by default does not copy files")
+    elif copy == CopyFileDetection.ALL or copy == CopyFileDetection.LOADED_MODULES:
+        # for those migrating, who only set --copy all/auto but don't have --fast set.
+        fast = True
+
```

Comment:
Can we disallow `--copy` and `--fast` at the same time?

Also if `--fast` is passed in, I'll want to raise a warning message that points people to docs that explain "how to use --copy".

Reply:
adding an exception, but no docs to point to as yet.  can update after those have been published.